src/style.css

```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
```

Comment:
Wrong-way, instead, give these `font-faces` the same `font-family`, and set different `font-weights` (i.e. 300 and 900)

Reply:
This allows you to mention `font-family` and `font-weight` for the parent once, and then only change `font-weight` for its children, `font-family` remains the same